### PR TITLE
Add orange box competition takeover and engage page

### LINKED ISSUE #7089: [Takeover] Orange box] - Build takeover and engage page
## Requirements

Please build a takeover and an engage page based on this [brief](https://docs.google.com/spreadsheets/d/19aWd11o6y8ZDHS6NdK4wHDRCsSRAeiTsDdkLSM5ZAAA) and [design](2453):
 
- [x] build takeover
- [x] build engage page
- [x] add takeover to `/takeovers/index.html`
- [x] add takeover to `/index.html`
- [x] add engage page to `/engage/index.html`
- [ ] test the engage page on [https://socialdebug.com/](https://socialdebug.com/) or [http://debug.iframely.com/](http://debug.iframely.com/) and [https://cards-dev.twitter.com/validator](https://cards-dev.twitter.com/validator)

### PR DESCRIPTION
**Pull request by deadlight**
## Done

- Takeover and engage page

[brief](https://docs.google.com/spreadsheets/d/19aWd11o6y8ZDHS6NdK4wHDRCsSRAeiTsDdkLSM5ZAAA) | [design](2453) | [Engage copy](https://docs.google.com/document/d/19-Fpi-4XO_pGwukz_ZjSFhmraV58Os42AN4UJJq8FgI/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/ and refresh until you see the "Quick, easy, on-prem Kubernetes" - test all viewports
- Click the CTA to see the engage page - test all viewports
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

## Issue / Card

Fixes #7089

### CODE REVIEW

File: templates/engage/orangebox.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+wrapper_template: "engage/_base_engage_markdown.html"
+context:
+     title: "Enter to win zero-ops Kubernetes in an Ubuntu Orange Box"
+     meta_description: "Enter to win zero-operations Kubernetes in an Ubuntu Orange Box with MicroK8s, a fast, secure, upstream Kubernetes distribution from Canonical."
+     meta_image: https://assets.ubuntu.com/v1/e4c3046f-orangebox-meta.png
+     meta_copydoc: https://docs.google.com/document/d/1xJVtEHr5ROGNzDblPDNVNk4DcX5HGcXL7gyrRPKcUdE
```
Comment (anthonydillon) [inline templates/engage/orangebox.md:7]: This is the "An introduction to Anbox Cloud" copydoc

Comment (deadlight) [inline templates/engage/orangebox.md:7]:
```suggestion
     meta_copydoc: https://docs.google.com/document/d/19-Fpi-4XO_pGwukz_ZjSFhmraV58Os42AN4UJJq8FgI/edit
```

**Comment by deadlight** [comment]
@anthonydillon whoops. Sorry about that. Fixed the link

**Comment by anthonydillon** [comment]
Of course, needs a link to the T&Cs when available.

**Comment by Khutch25** [comment]
Posted some comments in the copy doc - https://docs.google.com/document/d/19-Fpi-4XO_pGwukz_ZjSFhmraV58Os42AN4UJJq8FgI/edit#

**Comment by spencerbygraves** [comment]
@deadlight the image is a bit smaller than intended. Can the width be 395px on both pages please. Thanks.

**Comment by spencerbygraves** [comment]
@deadlight thanks 👍

File: templates/engage/index.html [hunk 2]
```diff
@@ -968,11 +968,19 @@ <h1>Ubuntu case studies, whitepapers and webinars</h1>
       {% include "engage/_article-card.html" %}
     {% endwith %}
 
+<<<<<<< HEAD
```
Comment (anthonydillon) [inline templates/engage/index.html:971]: Conflict

**Comment by Khutch25** [comment]
Amber added some additional comments in on feedback from the UK team in the copy doc - https://docs.google.com/document/d/19-Fpi-4XO_pGwukz_ZjSFhmraV58Os42AN4UJJq8FgI/edit?usp=sharing.

Since more design work is needed, not sure if it is worth making these changes or putting it on hold until that work can be done, hopefully in that iteration starting on the 27th, but I'll leave that with you all!

**Comment by deadlight** [comment]
Brief changed, closing

**Pull request closed without merging**